.github/workflows/pr-percy-prepare-label.yml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   copy_artifact:
     name: Copy changed files to GHA artifact
-    if: "contains(toJson(github.event.pull_request.labels.*.name), 'Review: Percy needed')"
+    if: "(github.event.action == 'synchronize' && contains(toJson(github.event.pull_request.labels.*.name), 'Review: Percy needed')) || (github.event.action == 'labeled' && github.event.label.name == 'Review: Percy needed')"
```

Comment:
Is there an "else"? Not in scope of this PR, but I wonder if there is a way for logs to tell us "skipping because label XYZ, etc", to have better understing why something runs and why it's skipped.

Reply:
@bartaz We could add an else - the only consideration is that because this is the only job in the workflow and it is skipped if this case is not met, that shows the entire workflow as skipped, which may be helpful:

![image](https://github.com/canonical/vanilla-framework/assets/46915153/f34d8e38-f6dd-4d6d-9649-6913432e2164)

If we add a job that *does* run in the opposite case, then this workflow will always run, and appear as a green check in PRs & the workflow list.